Assets/Project/Scripts/Editor/StageDataEditor.cs

```diff
@@ -60,6 +70,15 @@ public override void OnInspectorGUI()
 
             serializedObject.ApplyModifiedProperties();
             _numOfAttackableBottles = GetAttackableBottles()?.Count() ?? 0;
+            // ギミックの個数が増えたとき、増えたギミックはデフォルト値(Tornado)に設定する
+            if ((_src.GimmickDatas?.Count() ?? 0) > _numOfGimmicks) {
+                var newGimmcikIndices = Enumerable.Range(0, _gimmickDatasProp.arraySize)
+                    .Where(index => index >= _numOfGimmicks);
+                foreach (var gimmickIndex in newGimmcikIndices) {
+                    _gimmickDatasProp.GetArrayElementAtIndex(gimmickIndex).FindPropertyRelative("type").enumValueIndex = (int)EGimmickType.Tornado;
+                }
```

Comment:
`var newGimmcikIndices` のスペルが酷すぎることは置いておいて・・・以下のような感じでどうでしょう．
```suggestion
                Enumerable.Range(0, _gimmickDatasProp.arraySize)
                    .Where(index => index >= _numOfGimmicks).ToList()
                    .ForEach(index =>
                    {
                        _gimmickDatasProp.GetArrayElementAtIndex(index).FindPropertyRelative("type").enumValueIndex = (int)EGimmickType.Tornado;
                    });
```

改行がうざければ，以下でも良いです

```suggestion
                Enumerable.Range(0, _gimmickDatasProp.arraySize)
                    .Where(index => index >= _numOfGimmicks).ToList()
                    .ForEach(index => _gimmickDatasProp.GetArrayElementAtIndex(index).FindPropertyRelative("type").enumValueIndex = (int)EGimmickType.Tornado);
```

Reply:
linqに`ForEach`自体あるのか、盲点や。。。